packages/flutter/lib/firebase.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service exposed to Flutter apps that implements a subset of the Firebase
+/// API.
+///
+/// This library will probably be moved into a separate package eventually.
+library firebase;
```

Comment:
IMHO we should create a new `packages/firebase` to house this code.  Eventually we'll want to scale this up to wrap a large number of third-party SDKs.  We won't want to have them all in a monolithic flutter package...